docs/sources/explore/_index.md

```diff
@@ -87,6 +87,20 @@ To open the content outline:
 
 You can then click on any panel icon in the content outline to navigate to that panel.
 
+### Filter logs in content outline
+
+When using Explore with logs, you can filter the logs in the content outline. You can filter by log level, currently supported for Elasticsearch and Loki data sources. To select multiple filters, hold down the command key on Mac or the control key on Windows while clicking.
```

Comment:
Should we mention that it's availbe only if the log volume histogram is shown and contains multiple levels?

Reply:
Im not sure if this is digging into specifics too much, but a short nudge in how to see data if they're not might be helpful to new folks. 

```
{{% admonition type="note" %}}
Log levels will only show if the datasource supports the log volume histogram and contains multiple levels. Additionally, the query to the datasource may have to format the log lines to see the levels. For example, in loki, the `logfmt` parser commonly will display log levels.
{{% /admonition %}}

```

Reply:
@gelicia this is great! thanks for the suggestion. I think this will be helpful and it's not digging into specifics too much